CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 * None
 
 ### Enhancements
-* None
+* Opening a realm with invalid schemas will throw a `SchemaParseError` rather than an `AssertionError` or `Error`.
```

Comment:
```suggestion
* Opening a Realm with invalid schemas will throw a `SchemaParseError` rather than an `AssertionError` or `Error`. ([#5198](https://github.com/realm/realm-js/issues/5198))
```

Reply:
Thanks!